ksp_plugin_test/plugin_compatibility_test.cpp

```diff
@@ -18,14 +20,29 @@
 namespace principia {
 namespace interface {
 
+using astronomy::operator""_TT;
+using astronomy::TTSecond;
+using astronomy::date_time::DateTime;
+using astronomy::date_time::operator""_DateTime;
 using base::not_null;
 using base::OFStream;
 using base::PullSerializer;
 using base::PushDeserializer;
 using ksp_plugin::Plugin;
+using quantities::Speed;
 using testing_utilities::ReadLinesFromBase64File;
 using testing_utilities::ReadLinesFromHexadecimalFile;
+using ::testing::AllOf;
+using ::testing::ElementsAre;
+using ::testing::Eq;
+using ::testing::HasSubstr;
+using ::testing::IsFalse;
+using ::testing::IsTrue;
+using ::testing::Not;
 using ::testing::NotNull;
+using ::testing::Pair;
+using ::testing::internal::CaptureStderr;
```

Comment:
Let's not access the internals, please.  It's going to break whenever Google feel like changing their implementation.

The same effect could be achieved in a portable manner with `EXPECT_EXIT`.

Reply:
Not really, because then it needs to be a death test and this makes it tricky (and unreadable) to have more complicated tests like the Reach one.

The proper way to do this would be to take advantage of the fact that we have decided to diverge our glog and have it be instrumentable, instead of reaching all the way to standard error (either directly or even more circuitously via EXPECT_EXIT).

Reply:
A `LogSink` does the job.